draft-ietf-httpbis-http2.xml

```diff
@@ -3333,9 +3333,11 @@ HTTP2-Settings    = token68
           Promised requests MUST be cacheable (see <xref target="RFC7231" x:fmt=","
           x:rel="#cacheable.methods"/>), MUST be safe (see <xref target="RFC7231" x:fmt=","
           x:rel="#safe.methods"/>) and MUST NOT include a request body. Clients that receive a
-          promised request that is not cacheable, unsafe or that includes a request body MUST
-          reset the stream with a <xref target="StreamErrorHandler">stream error</xref> of type
-          <x:ref>PROTOCOL_ERROR</x:ref>.
+          promised request that is not cacheable, unsafe or that indicates the presence of a request

```

Comment:
"...that is not cachable, unsafe..." implies "is not unsafe." It's obvious in the context of the prose above, but "...is not cachable, is unsafe..." is less ambiguous.